Tasks/DotNetCoreInstallerV0/dotnetcoreinstaller.ts

```diff
@@ -90,13 +90,13 @@ class DotnetCoreInstaller {
     private async downloadAndInstall(downloadUrls: string[]) {
         let downloaded = false;
         let downloadPath = "";
-        for (var i = 0; i < downloadUrls.length; i++) {
+        for (const url of downloadUrls) {
             try {
-                downloadPath = await toolLib.downloadTool(downloadUrls[i]);
+                downloadPath = await toolLib.downloadTool(url, url.split('/').pop());
```

Comment:
lib's recommendation is to not use filename

Reply:
Changed the logic to select zip/tar